wpilibj/src/athena/java/edu/wpi/first/wpilibj/DigitalInput.java

```diff
@@ -33,7 +33,7 @@ public DigitalInput(int channel) {
     checkDigitalChannel(channel);
     m_channel = channel;
 
-    m_handle = DIOJNI.initializeDIOPort(DIOJNI.getPort((byte)channel), true);
+    m_handle = DIOJNI.initializeDIOPort(DIOJNI.getPort((byte) channel), true);
```

Comment:
Always funny to see differences between languages. In C#, the no-space version is idiomatic.